# Identification of Cardiovascular Disease-related Genes Based on Genome-wide Blood Transcriptome

## Abstract
**Background:** Inference of co-expression network and identification of disease-related modules and gene sets can help understand disease-related molecular pathophysiology. We aimed to identify a cardiovascular disease (CVD)-related transcriptomic signature, specifically in peripheral blood tissue, based on differential expression (DE) and differential co-expression (DcoE) analyses.

**Methods:** Publicly available CVD datasets derived from blood samples were integrated to establish a co-expression network. Weighted gene co-expression network analysis was used to construct modules which included genes with highly correlated expression values.

**Results:** Twenty-one modules were established, among which the turquoise and yellow modules were selected based on the DE and DcoE criteria. For each module, geneâ€“gene interactions with extremely high correlation coefficients were individually selected under two conditions. Genes displaying a significantly changed number of edges were selected. Finally, 11 genes in the turquoise module and genes in the yellow module were identified as potential CVD-related genes.

**Conclusion:** Our study identified gene sets and finger-countable genes that were dysregulated in CVD blood samples. These findings may contribute to the understanding of CVD pathophysiology.

## Data



## Analysis
Comparison of disease-related transcriptomic signature among three blood CVD datasets. Each point indicates a gene. Y and X axes are logFC between two statuses described in the form of (A/B).

